core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.ts

```diff
@@ -91,7 +94,6 @@ export class CkEditorCopyContentService {
         ...Array.from(currentDescendant.childNodes)
       ];
     }
-
```

Comment:
Unneeded change?

Reply:
Can revert this if necessary especially since it doesn't affect output.

Reply:
I mean I don't see a reason for this change, if you have one it is fine to keep it here.

Reply:
Following: A PR should have minimal changes and should not include unwanted changes (increases the diff size and changes the history of the line) & the newline here is used for presenting visual blocks `descendants loop` and other checks.

I would suggest undoing these changes! :) [Thanks @vojtechjelinek for raising it!]

Reply:
Done.

Reply:
I have fixed it! I get the point about unnecessary changes.